File: 0kth-largest-element-in-an-array/0kth-largest-element-in-an-array.cs
public class Solution {
    public int FindKthLargest(int[] nums, int k) {
        
        PriorityQueue<int,int> minheap=new PriorityQueue<int,int>();
        
        for(int i=0;i<nums.Length;i++)
        {
            minheap.Enqueue(nums[i],nums[i]);
            if(minheap.Count>k)
            {
                minheap.Dequeue();
            }
        }
        
        return minheap.Peek();
    }
}

File: 0kth-largest-element-in-an-array/0kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        
        PriorityQueue<Integer> queue=new PriorityQueue<>();
        
        for(int num : nums)
        {
            queue.add(num);
            while(queue.size()>k)
            {
                queue.remove();
            }
        }
        
        return queue.peek();
        
    }
}